src/QueryM/Ogmios.purs

```diff
@@ -209,7 +303,8 @@ instance Show ChainOrigin where
 
 -- | A point on the chain, identified by a slot and a block header hash
 type ChainPoint =
-  { slot :: Slot
+  { slot :: BigInt -- I think we need to use `BigInt` here, 18446744073709552000
```

Comment:
Maybe we should change Slot to wrap BigInt instead. Both node uses Integer, ogmios U64 and CSL goes with U32.
I guess similar thing wrt `Epoch`. At least we could reclaim our newtype wrappers and make the query result 
more safe

Reply:
Alternatively, we could set up another layer of newtype wrappers along with descriptions/docstrings for "securing" ogmios schema.
Idk.
I feel like we're missing an overarching unified approach to 'transporting' types between services.